docs/dapps/nfts/intro-to-erc721s.md

```diff
@@ -0,0 +1,140 @@
+
+Once the image and metadata files are ready, we can prepare to deploy a smart contract.
+
+## Preparing Your Environment
+
+### MetaMask Extension
+
+You will need the MetaMask Extension installed on whatever browser you are using to be able to fund the deployment of the smart contract. If you have not done so already, download MetaMask and [add the Fuji network to MetaMask](https://support.avax.network/en/articles/6224787-how-to-connect-metamask-to-the-fuji-testnet). Create or import a Fuji account as necessary.
+
+### Getting Testnet Funds
+
+Because we are deploying on the Fuji Network, you will need to get AVAX on the Fuji network. If you visit the [Avalanche Faucet](https://faucet.avax.network/), you can request up to 2 Fuji AVAX per day. Please enter the C Chain address of the account linked to your MetaMask in the previous step. 
+
+![Avalanche Faucet](intro-to-erc721s/1-faucet.png)
+
+## Creating the Smart Contract
+
+To create the smart contract, we are going to use [Open Zeppelin](https://docs.openzeppelin.com/). Open Zeppelin is a key tool for building smart contracts quickly and easily. While we are only scratching the surface in this tutorial, ample documentation is available on their website for you to read when you want to build more complex contracts.
+
+Open Zeppelin provides a [Contract Wizard](https://docs.openzeppelin.com/contracts/4.x/wizard) that will build out ERC contracts. To avoid any complex coding environments, we will use this to create our ERC721 contract.
+
+![Contract Wizard](intro-to-erc721s/2-contract-wizard.png)
+
+Select `ERC721` on the Contract Wizard to get started. This will create the contract in the [Solidity programming language](https://docs.soliditylang.org/en/v0.8.15/).
+
+As you can see, the template contract is bare-boned. We will fill out the information in the left panel to auto-populate it into our contract. Make sure you change to the ERC721 tab as you get started to make the proper contract.
+
+The Wizard auto-fills in a name and symbol for your NFT collection, which we will modify here. I am naming it `Photography` and giving it the symbol `FOTO`. If you chose your own files to use during this tutorial, you can choose a relevant name and symbol for your collection.
+
+The Base URI field listed here is the URL of the metadata folder uploaded to Pinata(for example, ours is `https://gateway.pinata.cloud/ipfs/QmYdWxbiwsfsYcW1CYQPgYujAc9FMLPG3fgFcxFskbSsFa`). Paste that into the Base URI field. After the Wizard adds our variables to the template, our contract should look like this:
+
+![Contract Wizard Populated](intro-to-erc721s/3-wizard-populated.png)
+
+Next, we will want to check the `Mintable` and `Auto Increment Ids` boxes. This will populate a mint function into our template that would handle the incrementation of token Ids on mint if we had more than one NFT in our collection. We still want it to auto-assign our 1 NFT, so we will check it.
```

Comment:
Consider using contractions to make the text read more casually.

Ex: "We'll want to check..." instead of "We will want to check..." and "we'll check it" instead of "we will check it"